src/Views/MultiStepForm.jsx

```diff
@@ -267,7 +268,7 @@ function MultiStepForm(props) {
 
     return (
         <>
-            {step != 6 && <BackButton stringContent={stringContent} type={'button'} onClick={handlePrev} text={backButtonText(step)}/>}
+            {step != 6 && <BackButton stringContent={stringContent} type={'button'} data-test="backBtn" onClick={handlePrev} text={backButtonText(step)}/>}
```

Comment:
@SamiraMSadat can we update to something like `data-ga` or maybe `data-analytics`?

Unless you need to use that.. if thats the case i have that data attribute on the back button component

Reply:
It can be changed to anything, so sure